### PR TITLE
React-big-calendar add missing component type

### PR DESCRIPTION
**Pull request by fekete965**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:  N/A

Unfortunately, there is no documentation regarding the component property types. I am using react-big-calendar and I observed the properties that some of the custom components received. Unfortunately, the current type definition is not correct. The component property type was already defined, but it was probably forgotten to be added to the component type declaration.

I've also found a couple of imports which are not used, those have been removed.
I also found an interface which was probably a "starting point" of the final interface which is actually being used.

Any feedback is more than welcome, and please let me know if I have missed something regarding the process of contributing to this repo. Thanks

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-big-calendar ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-big-calendar@latest/))
was missing the following properties:
1. components
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 97b897e3f1aad1ba874ee0a7991238cc3bd4f297
</p>

File: types/react-big-calendar/index.d.ts [hunk 1]
```diff
@@ -536,24 +536,19 @@ export class Calendar<TEvent extends object = Event, TResource extends object =
     CalendarProps<TEvent, TResource>
 > {}
 
-export interface components {
-    timeSlotWrapper: React.ComponentType;
-    dateCellWrapper: React.ComponentType;
-    eventWrapper: React.ComponentType<Event>;
-}
```
Comment (michalak111) [inline types/react-big-calendar/index.d.ts:543]: not sure this should be removed, this is an interface for the object library is exporting

https://github.com/jquense/react-big-calendar/blob/537c6f3f08344191588f2ab3ad52667de7ee261f/src/index.js#L4-L8

Comment (fekete965) [reply types/react-big-calendar/index.d.ts:543]: Good point. I did not see that definition 👍

Comment (fekete965) [reply types/react-big-calendar/index.d.ts]: @michalak111 , I have reverted this change.

**Review by michalak111 — APPROVED** [review]

**Comment by fekete965** [comment]
Ready to merge

**Comment by fekete965** [comment]
@michalak111, nothing has changed, I've only rebased the PR.

**Review by michalak111 — APPROVED** [review]

**Comment by fekete965** [comment]
Ready to merge

**Pull request merged**